File: finger_exercises/largest_odd.c
#include <stdio.h>

int main(void){
    int num[10];
    int ans = 0;
    int i = 0;

    while (i < 10){
        printf("Please enter an integer: ");
        scanf("%d", &num[i]);
        if ((num[i] % 2 != 0) && (num[i] > ans)){
            ans=num[i];
            printf("%d\n", num[i]);
        }
        i++;
    }
    if (ans != 0){
        printf("The largest odd number is: %d\n", ans);
    }
    else{
        printf("No odd number.\n");
    }
}

File: finger_exercises/largest_odd.py
"""Write a program that asks the user to input 10 intergers, and prints
the largest odd number that was entered. If no odd number was entered, print
a message to that effect (Use a while loop)"""

ans = False
i = 0

while (i < 10):
    num = int(input("Enter an integer: "))
    if num % 2 != 0 and num > ans:
        ans = num
    i+=1

if ans:
    print(f"The largest odd number is {ans}.")
else:
    print("No odd number was entered.")